schnorr_fun/src/frost.rs

```diff
@@ -504,6 +551,20 @@ impl<H: Digest<OutputSize = U32> + Clone, NG: NonceGen> Frost<H, NG> {
         rng
     }
 
+    /// Create a keygen id to seed a keygen
+    /// TODO
+    pub fn create_keygen_id(&self, point_polys: &Vec<(Scalar<Public>, Vec<Point>)>) -> [u8; 32] {
```

Comment:
why doesn't this just take a `KeyGen`? I see now that we are storing a `keygen_id` redundantly in `KeyGen`. It should just be `pub fn keygen_id(&self, keygen: &KeyGen) -> [u8;32]` and we shoudn't store it in `KeyGen`.